src/reactive-graphql.ts

```diff
@@ -359,7 +359,7 @@ function resolveField(
       null // that would be the info
     );
 
-    if (resolvedValue instanceof Observable) {
+    if (isObservable(resolvedValue)) {
```

Comment:
Hmm, I don't really see the benefit of [duck typing](https://github.com/graphql/graphql-js/blob/fd308ceb72a90a3903411a830da46adcd1156177/src/jsutils/isPromise.js) instead of using `instanceof`, could you expand on your suggestion?

Reply:
In some cases, the user thinks he is returning a Promise but it's not treated as it by `reactive-graphql`. For instance, the user uses a lib that leverages other implementation of Promise than the native one.

For example, [Sequelize uses](https://github.com/sequelize/sequelize/blob/17a4c1566b48e1b25f336523cd9924db14c936f2/lib/promise.js) [bluebird](http://bluebirdjs.com/docs/getting-started.html) and returns instance of `bluebird.Promise` that will not be `instanceof global.Promise`.

As Promise became am es6 standard, this case have les and less chance to happen, but still.

However is even more true for Observable, as many implementations exist.

Reply:
I was not aware that these implementations cohabit 🤔